tests/conftest.py

```diff
@@ -93,3 +93,29 @@ def adr_service_query(request, pytestconfig: pytest.Config) -> Service:
         tmp_service._container.save_config()
     tmp_service.start(create_db=False, exit_on_close=True, delete_db=False)
     return tmp_service
+
+
+@pytest.fixture
+def adr_template_json(request, pytestconfig: pytest.Config) -> Service:
+    use_local = pytestconfig.getoption("use_local_launcher")
```

Comment:
I would avoid creating new fixtures just to add a few templates. If you are only testing new template examples, please add them to the existing database and use the existing fixtures instead.